.circleci/config.yml

```diff
@@ -9,13 +9,19 @@
 
 ## IMPORTANT
 # If you change the `docker_image` version, also change the `cache_key` suffix
-var_1: &docker_image angular/ngcontainer:0.1.0
-var_2: &cache_key rules_typescript-{{ checksum "yarn.lock" }}-0.1.0
+var_1: &docker_image angular/ngcontainer:0.3.0
+var_2: &cache_key rules_typescript-{{ checksum "yarn.lock" }}-0.3.0
 var_3: &setup-bazel-remote-cache
   run:
     name: Start up bazel remote cache proxy
     command: ~/bazel-remote-proxy -backend circleci://
     background: true
+# Move node binaries out of the way to enforce that Bazel uses
+# only the hermetic ones it downloads
+var_4: &hide_node_and_yarn_local_binaries
```

Comment:
there's a nicer pipe syntax; switch this to

```
var_4: &hide_node_and_yarn_local_binaries
  run:
    name: Move node, npm, and yarn binaries
    command: |
      sudo mv /usr/local/bin/node /usr/local/bin/node_
      sudo mv /usr/local/bin/npm /usr/local/bin/npm_
      sudo mv /usr/local/bin/yarn /usr/local/bin/yarn_
```

Reply:
Thanks